tests/test_VariableDayLength

```diff
@@ -0,0 +1,16 @@
+import unittest
```

Comment:
Nitpick here: this file is missing the `.py` extension. Just adding that to the file name would bring it in line with other files in the folder.